### ISSUE TITLE
Taskbar has once again vanished from a display after it was powered off overnight

### ISSUE THREAD

**Issue by kode54** [LBCrion/sfwbar#198]
It only happened once this time, on labwc-git with the wlroots 0.18 draft PR. Not sure I can reproduce it, even.

**Comment by LBCrion** [comment]
Was it the entire bat that didn't appear or just the taskbar (the window
list)?

**Comment by kode54** [comment]
Ah, sorry. I meant the entire bar didn't appear. I simply restarted it.

And I haven't been able to reproduce whatever caused that disappearance, either. Let me try again now... Nope. Powering it off again didn't make it disappear. Weird. Maybe it was a one-off occurrence.
